integration_tests/dbt_project.yml

```diff
@@ -1,44 +1,49 @@
 name: 'social_media_rollup_integration_tests'
-version: '0.3.0'
+version: '0.4.0'
 profile: 'integration_tests'
 config-version: 2
 
 vars:
   twitter_organic_source:
-    twitter_organic_schema: social_media_rollup_integration_tests
-    twitter_organic_account_history_identifer: "twitter_organic_account_history_data"
-    twitter_organic_organic_tweet_report_identifer: "twitter_organic_organic_tweet_report_data"
-    twitter_organic_tweet_identifer: "twitter_organic_tweet_data"
-    twitter_organic_twitter_user_history_identifer: "twitter_organic_twitter_user_history_data"
+    twitter_organic_schema: social_media_rollup_integration_tests_1
+    twitter_account_history_identifier: "account_history"
+    twitter_organic_tweet_report_identifier: "organic_tweet_report"
+    twitter_tweet_identifier: "tweet"
+    twitter_twitter_user_history_identifier: "twitter_user_history"
```

Comment:
These changes were made to ensure the union_data macro was working as expected. We cannot effectively test the union_data macro when the seed files are named different from the expected source names.